scipy/stats/_continuous_distns.py

```diff
@@ -7494,8 +7500,11 @@ def _logcdf(self, x, a):
     def _ppf(self, q, a):
         return pow(q, 1.0/a)
 
-    def _sf(self, p, a):
-        return -sc.powm1(p, a)
+    def _sf(self, x, a):
+        return -sc.powm1(x, a)
```

Comment:
I renamed `p` to `x` here to be in line with the naming convention in the rest of the file.